.packit.yaml

```
@@ -189,3 +189,13 @@ jobs:
       - fedora-branched
       - epel-9
       - epel-10
+
+  # Run missing tests identified in Connect
+  - <<: *test-base
+    <<: *internal
+    <<: *require-full-tests
+    identifier: internal-fusa
+    fmf_url: "https://gitlab.cee.redhat.com/automotive/tests/all.git"
+    tmt_plan: '/tmt/plans/fusa/tests'
+    targets:
+      - centos-stream-9-x86_64
```

```suggestion
      - epel-9
```

Would also be preferred if this was moved above the downstream jobs, preferably near the other internal jobs.